docs/reference/setup/important-settings/heap-size.asciidoc

```diff
@@ -49,7 +49,8 @@ caches, but the less memory it leaves available for the operating system to use
 for the filesystem cache. Also, larger heaps can cause longer garbage
 collection pauses.
 
-Here are examples of how to set the heap size via the jvm.options file:
+Here are examples of how to set the heap size via a `jvm.options.d/heap.options`
```

Comment:
Similar to the other discussion in this PR, maybe we should use a different name here so that users don't think that `heap.options` is a special name?

Reply:
I removed the explicit filename.